integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java

```diff
@@ -119,6 +121,14 @@ public static void main(String[] args) {
     }
     CommonUtils.PROCESS_TYPE.set(CommonUtils.ProcessType.CLIENT);
     MetricsSystem.startSinks(conf.get(PropertyKey.METRICS_CONF_FILE));
+    if (conf.getBoolean(PropertyKey.FUSE_WEB_ENABLED)) {
+      FuseWebServer mWebServer = new FuseWebServer(
```

Comment:
local variable doesn't need to be `mName`
change to `webServer`?

Reply:
Sure, thanks for remind me.